lib/survey_gizmo/resource.rb

```diff
@@ -114,20 +112,23 @@ def handle_route!(key, interpolation_hash)
       private
 
       # Convert a [Hash] of params and internal surveygizmo style filters into a query string
-      def filters_to_query_string(filters = {})
-        return '' unless filters && filters.size > 0
+      def filters_to_query_string(params = {})
+        return '' unless params && params.size > 0
+
+        url_params = {}
+        filters = params.delete(:filters) || []
+        filters = [filters] unless filters.is_a?(Array)
```

Comment:
This is the only even marginal functional change - you can now pass a single hash as a filter instead of always having to pass an array of hashes.